lib/cloudfunc.js

```diff
@@ -218,6 +218,12 @@
                     mode        : '--- --- ---'
                 });
             }
+
+            // OSC_CUSTOM_CODE hide all dot files
+            function filesWithoutDots(value) {
+                return value.name.charAt(0) !== '.';
+            }
+            files = files.filter(filesWithoutDots);
```

Comment:
Not sure why it did that. I retyped it by hand and it worked...

https://github.com/OSC/cloudcmd/pull/6/commits/6970b7dd29561bf8d21a9413f5a0b80a8c917fdd